etc/config/c++.amazon.properties

```diff
@@ -926,7 +843,7 @@ group.avr.compilers=avrg454:avrg464:avrg540:avrg920:avrg930:avrg1030:avrg1100
 group.avr.groupName=AVR GCC
 group.avr.baseName=AVR gcc
 group.avr.isSemVer=true
-group.avr.supportsBinary=true
```

Comment:
The parent group already sets this to true

Reply:
:+1: